### ISSUE TITLE
fix/update-input-design

### ISSUE THREAD

**Issue by max8392** [pauloamgomes/contentful-better-slugs#17]
Many thanks for the extension. I updated the design of the input to match the current design of Contenful. If you want I can submit my pull request to you. 

css replace 
```

.iframe {
  height: 50px,
}
.container {
  display: flex;
  flex-wrap: nowrap;
  align-items: center;
  margin: 3px;
}
.container input {
  -webkit-appearance: textfield;
  background-color: #fff;
  border: 1px solid #cfd9e0;
  border-radius: 6px;
  -webkit-box-shadow: inset 0 2px 0 #e1e4e833;
  box-shadow: inset 0 2px 0 #e1e4e833;
  color: #414d63;
  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica,
    Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
  font-size: 0.875rem;
  margin: 0;
  max-height: 2.5rem;
  outline: none;
  padding: 0.65625rem;
  width: 100%;
}
.container input:focus {
  border: 1px solid #036fe3;
  -webkit-box-shadow: 0 0 0 3px #98cbff;
  box-shadow: 0 0 0 3px #98cbff;
  outline: none;
}
```

**Comment by pauloamgomes** [comment]
Hi @max8392 , thanks for that, yeah, please go ahead and create a PR.

**Comment by max8392** [comment]
> Hi @max8392 , thanks for that, yeah, please go ahead and create a PR.

Hello, Perfect, could you give me push rights to my branch? Please

**Comment by pauloamgomes** [comment]
Think you can just fork and then do the PR, that is not working?

**Comment by max8392** [comment]
> Think you can just fork and then do the PR, that is not working?

Sorry, this is my first time doing this sort of thing here. Here is my PR is available. Thanks very much

**Comment by pauloamgomes** [comment]
closed by #18
